samples/legacy_support/All_ID_Mapping_Original.csv

```diff
@@ -1005,7 +1005,7 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 1003,Movement and Maneuver : Aviation Fixed Wing : Reconnaissance : Original,S-G-UCVFR-,S-G-UCVFR-,,,,TRUE,Point,,,
 1004,Movement and Maneuver : Combat,S-G-UC----,10120900,,,,,Point,,,
 1005,Movement and Maneuver : Infantry,S-G-UCI---,10121100,,,,TRUE,Point,,,
-1006,Movement and Maneuver : Infantry : Capability : Signals Intelligence : Mobility : Airborne,S-F-GR----,10121100,10571,10012,,TRUE,Point,,,
+1006,Movement and Maneuver : Infantry : Capability : Signals Intelligence : Mobility : Airborne,S-F-GR----,10121100,10521,10012,,TRUE,Point,,,
```

Comment:
This is an ill advised change to the mapping.  Land Unit Sector One modifier 52 is an R for "Radiological", not "Ranger",  With this requested change JMSML will produce a legacy symbol name of "Movement and Maneuver : Infantry : Radiological : Airborne", not the correct name.  The recommended fix for this is therefore to create a new SVG and not map to the 2525D "R" modifier.